src/lib/networksRegistry.js

```diff
@@ -14,24 +14,8 @@ const testNetwork = {
   compilerUrl: 'https://compiler.aepps.com',
 };
 
-let networks = process.env.NODE_ENV === 'production' ? [
-  mainNetwork,
-  testNetwork,
-] : [
-  testNetwork,
-  mainNetwork,
-];
-
-export const defaultNetwork = process.env.VUE_APP_NETWORK_NAME ? {
```

Comment:
How do I change the networks/URLs' during development outside docker ?

Reply:
I would add a network manually and switch to it. Base aepp would remember it unless the state is reset. Do you need something more permanent?

We were developing Base aepp on testnet without running a local node/mdw. The last time I tried to set up mdw for e2e tests it didn't work well.
https://github.com/aeternity/ae_mdw/issues/1336

Reply:
Well, I actually tested and developed (some very small features) locally using local node/network. After all we're looking for extension point to use the wallet in custom networks.